MainDemo.Wpf/IconPackViewModel.cs

```diff
@@ -63,7 +63,7 @@ private void Search(object obj)
         private void CopyToClipboard(object obj)
         {
             var kind = (PackIconKind?)obj;
-            Clipboard.SetText($"<materialDesign:PackIcon Kind=\"{kind}\" />");
+            Clipboard.SetDataObject($"<materialDesign:PackIcon Kind=\"{kind}\" />");
```

Comment:
We probably do want to take this change out of this PR since it is already included in #749